benchmarks/ai-benchmark/benchmark.py

```
@@ -2,32 +2,34 @@
+    body = json.dumps({
+        "model": model,
+        "messages": [{"role": "user", "content": prompt}],
+        "stream": stream,
+    }).encode()
+    req = urllib.request.Request(
+        url, data=body, headers={"Content-Type": "application/json"}
     )
-    r.raise_for_status()
+    # urlopen raises HTTPError on a non-2xx status and does not read the body
+    # up front, so lines arrive as the server sends them.
+    r = urllib.request.urlopen(req)
```

Ruff (0.16.0)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)